### DISCUSSION TITLE
Can't load any custom songs

### DISCUSSION THREAD

**Discussion by noskincz** [KadeArchive/Kade-Engine#562] [Q&A]
https://user-images.githubusercontent.com/84130850/119352410-f1f42d00-bca1-11eb-879f-9913a7f1fd00.mp4

any help ill take cuz its for my mod (btw yes i modified fresh easy ok)

**Comment by puyoxyz** [comment]
console output?

**Reply by noskincz** [reply]
idk how to look at it lol

**Reply by puyoxyz** [reply]
run kade engine like `lime run windows` instead of doing `lime build windows`
